test/filter_test.rb

```diff
@@ -16,7 +16,7 @@ def [](field)
   end
 
   def setup
-    @ldap    = GitHub::Ldap.new(:uid => 'uid')
+    @ldap    = GitHub::Ldap.new(options.merge(:uid => 'uid'))
```

Comment:
Unrelated, but this was trying to connect to the wrong host.

It's a smell that it needs to connect on initialization but that is a separate problem.